src/scss/views/widgets/_pill.scss

```diff
@@ -0,0 +1,59 @@
+.pill {
+  display: flex;
+  align-items: center;
+  width: fit-content;
+  height: 32px;
+  border-radius: $border-radius-large;
+  font-size: 14px;
```

Comment:
fine for now, but we should be using variables for font-sizes so that we don't have ea million